packages/back-end/src/init/config.ts

```diff
@@ -239,8 +231,10 @@ export function getConfigSegments(organization: string): SegmentInterface[] {
       id,
       ...d,
       organization,
-      dateCreated: null,
-      dateUpdated: null,
+      dateCreated: new Date(), //temporary
+      dateUpdated: new Date(), //temporary
+      // dateCreated: null,
+      // dateUpdated: null,
```

Comment:
This is an issue - the `BaseModel` type requires dateCreated and dateUpdated, but config defined resources do not have these.